pkg/applicationserver/applicationserver_test.go

```diff
@@ -110,23 +110,23 @@ func TestApplicationServer(t *testing.T) {
 		"brands.yml": []byte(`version: '3'
 brands:
 thethingsproducts:
-  name: The Things Products
-  url: https://www.thethingsnetwork.org`),
+	name: The Things Products
+	url: https://www.thethingsnetwork.org`),
 		"thethingsproducts/devices.yml": []byte(`version: '3'
 devices:
-  thethingsnode:
-    name: The Things Node`),
+	thethingsnode:
+		name: The Things Node`),
 		"thethingsproducts/thethingsnode/versions.yml": []byte(`version: '3'
 hardware_versions:
-  '1.0':
-    - firmware_version: 1.1
-      payload_format:
-        up:
-          type: javascript
-          parameter: decoder.js
-        down:
-          type: javascript
-          parameter: encoder.js`),
+	'1.0':
+		- firmware_version: 1.1
+			payload_format:
+				up:
+					type: javascript
+					parameter: decoder.js
+				down:
+					type: javascript
+					parameter: encoder.js`),
```

Comment:
This doesn't look quite right. We shouldn't be indenting YAML with tabs just because it happens to be in a Go file.